bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -882,6 +891,72 @@ def get_magnum_Matrix4_spot_a_T_b(self, a: str, b: str, tree=None) -> mn.Matrix4
         mn_transformation = mn.Matrix4.from_(rotation_matrix, translation)
         return mn_transformation
 
+    # TODO: Remove this function in PR#118
+    @staticmethod
+    def convert_transformation_from_sophus_to_magnum(
+        sp_transformation: sp.SE3,
```

Comment:
Could you please test that we can install sophus to the existing conda env? i am worried that this requires reinstall the entire env or something else

Reply:
These 2 lines of code will install sophuspy

```
pip uninstall sophus
pip install sophuspy==0.0.8
```

Thanks for pointing this out, I have also updated the `setup_instructions.md` & `issues.md` for this issue.